### ISSUE TITLE
Simplify `gp init`

### ISSUE THREAD

**Issue by svenefftinge** [gitpod-io/gitpod#728]
I use `gp init` regularly, but only as an easy way to create the two files (.yml, .dockerfile). The interactivity is more friction than useful, as it asks me about unimportant stuff. Having talked to other users about that, I learned that some don't use it at all for that reason and other found it confusing to get those questions asked, because they imply you did understand what they are about.

Long story short, I propose to make the command non-interactive by default.
It should simply create and open the two files. Also the two files should have uncommented example code with links to further documentation and guides. Not too excessive but e.g. the `.gitpod.Dockerfile` should have at least a typical `apt-get` example and the `.gitpod.yml` should have a task and a port.

**Comment by jankeromnes** [comment]
`gp init` is now non-interactive. Thanks for making it happen! 💯
